canonical-web-and-design/vanilla-framework
# Add code snippet icon modifier class

## Done

Added `.p-code-snippet__block--icon`, along with three additional classes: `.is-linux-prompt`, `.is-windows-prompt` and `.is-url`.

Fixes #3472 

## QA

- Open [demo](https://vanilla-framework-3487.demos.haus/docs/examples/patterns/code-snippet/code-snippet-icon)
- See that it matches the [design](https://app.zeplin.io/project/5fa270d8eee4dbb267e579f1/screen/5ffc2fdcb65e6a99b78f9d47)
